templates/api/knowledge-panels/environment/ecoscore/total.tt.json

```diff
@@ -1,4 +1,27 @@
-[% SET sum_of_bonuses_and_maluses = product.ecoscore_data.adjustments.production_system.value + product.ecoscore_data.adjustments.packaging.value + product.ecoscore_data.adjustments.threatened_species.value + product.ecoscore_data.adjustments.origins_of_ingredients.value %]
+[% USE sum_of_bonuses_and_maluses = 0;
```

Comment:
I don't think that work because if some adjusments are set, and others are not, you will sum the same value multiple times.

But ou can replace this whole part by just this:

						[% SET sum_of_bonuses_and_maluses = (product.ecoscore_data.adjustments.production_system.value || 0) + (product.ecoscore_data.adjustments.packaging.value || 0) + (product.ecoscore_data.adjustments.threatened_species.value || 0) + (product.ecoscore_data.adjustments.origins_of_ingredients.value || 0) %]

This suppresses the warnings.